src/components/Platform.jsx

```diff
@@ -13,24 +12,22 @@ function Points(props) {
   );
 }
 
-function Picture(props) {
+function Picture(name, image) {
   return(
     <div className="pure-g">
       <div className="pure-u-1 pure-u-md-1-3"></div>
       <div className="pure-u-1 pure-u-md-1-3">
-        <img alt={props.name + "'s avatar"} className="pure-img-responsive avatar" src={props.image}/>
+        <img alt={name + "'s avatar"} className="pure-img-responsive avatar" src={image}/>
```

Comment:
JS template string may be useful 
``${name}'s avatar``
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals